# initial_joint_angles: [0, -90, 90, 0, -90, 0] # positions in degrees
initial_joint_angles: [0, 0, -160, -20, 90, 0] 
driving_joint_angles: [0, 0, -160, -20, 90, 0] # positions in degrees
scan_position_joint_angles: [-90, -60, -30, -125, 90, 0] # positions in degrees
# dropping_position defines the pose of the gripper in the BASE_FRAME (x, y, z, roll, pitch, yaw) in degrees
dropping_position: [-0.35, 0.00, 0.85, 0.0, 30, 180]
